### PR TITLE
Version 1.4.0 | Add a UserRoleAssignment.applies_to_all field, becaus…

### PR DESCRIPTION
**Pull request by iloveagent57**
…e explicit is better than implicit.

### CODE REVIEW

File: edx_rbac/models.py [hunk 1]
```diff
@@ -59,9 +60,19 @@ class UserRoleAssignment(TimeStampedModel, metaclass=UserRoleAssignmentCreator):
...
     """
+    role_class = None
 
     user = models.ForeignKey(settings.AUTH_USER_MODEL, db_index=True, on_delete=models.CASCADE)
-    role_class = None
+
+    applies_to_all = models.BooleanField(
+        default=False,
+        null=False,
+        db_index=True,
```
Comment (brittneyexline) [inline edx_rbac/models.py:70]: curious why this field should be indexed? maybe i missed the reasoning from the last pr

**Review by brittneyexline — APPROVED** [review]

File: edx_rbac/models.py (hunk 1 above)
Comment (iloveagent57) [reply edx_rbac/models.py:70]: Hmm, yeah, it doesn't need to be.

File: edx_rbac/models.py [hunk 2]
```diff
@@ -59,9 +60,19 @@ class UserRoleAssignment(TimeStampedModel, metaclass=UserRoleAssignmentCreator):
     """
     Model for mapping users and their roles.
     """
+    role_class = None
 
     user = models.ForeignKey(settings.AUTH_USER_MODEL, db_index=True, on_delete=models.CASCADE)
-    role_class = None
+
+    applies_to_all = models.BooleanField(
```
[the remark this answers is not in the document]
Comment (iloveagent57) [reply edx_rbac/models.py:67]: `applies_to_all_contexts` maybe?

[the remark this answers is not in the document]
Comment (kellmarie) [reply edx_rbac/models.py]: Yeah I like that more.

**Review by kellmarie — APPROVED** [review]

**Pull request merged**